sdks/python/apache_beam/runners/portability/fn_api_runner/translations.py

```diff
@@ -818,7 +841,7 @@ def _eliminate_common_key_with_none(stages, context, can_pack=lambda s: True):
   # elimination, and group eligible KeyWithNone stages by parent and
   # environment.
   def get_stage_key(stage):
-    if len(stage.transforms) == 1 and can_pack(stage.name):
+    if len(stage.transforms) == 1 and int(can_pack(stage.name)) > 0:
```

Comment:
`can_pack(stage.name)` is equivalent (assuming no negative values) and more clear.

Reply:
Fixed.